packages/qunit-dom/lib/__tests__/does-not-have-tagname.ts

```diff
@@ -63,7 +63,7 @@ describe('assert.dom(...).doesNotHaveTagName()', () => {
 
       expect(assert.results).toEqual([
         {
-          message: 'Element <unknown> should exist',
+          message: 'Element <not found> should exist',
```

Comment:
Please revert the message changes back to <unknown>.
If it's something really needed, I'd prefer it to be a separate PR.

```suggestion
          message: 'Element <unknown> should exist',
```